includes/Story_Post_Type.php

```diff
@@ -727,4 +731,46 @@ public function change_default_title( $data ) {
 		}
 		return $data;
 	}
+
+	/**
+	 * Registers the demo admin menu page.
+	 *
+	 * @todo Remove
+	 *
+	 * @return void
+	 */
+	public function add_menu_page() {
+		add_submenu_page(
+			'edit.php?post_type=' . self::POST_TYPE_SLUG,
+			__( 'Demo', 'web-stories' ),
+			__( 'Demo', 'web-stories' ),
+			'edit_posts',
+			'web-stories-demo',
+			// This is not actually used.
+			// \Google\Web_Stories\Story_Post_Type::redirect_demo() will redirect to the story editor anyway.
+			'__return_null',
+			20
+		);
+	}
+
+	/**
+	 * Redirects the demo page to edit a new story.
+	 *
+	 * @return void
+	 */
+	public function redirect_demo() {
+		global $pagenow;
+
+		if ( 'edit.php' === $pagenow && isset( $_GET['page'] ) && 'web-stories-demo' === $_GET['page'] ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			wp_safe_redirect(
+				add_query_arg(
+					[ 'post_type' => self::POST_TYPE_SLUG ],
+					admin_url( 'post-new.php?web-stories-demo' )
```

Comment:
```suggestion
					admin_url( 'post-new.php?web-stories-demo=1' )
```
Don't this be?

Reply:
The value doesn't matter. The param simply needs to be set.

Reply:
It is must for consistency